orangecontrib/text/widgets/tests/test_owscoredocuments.py

```diff
@@ -297,6 +301,42 @@ def test_sort_table(self):
         data = [model.data(model.index(i, 0)) for i in range(model.rowCount())]
         self.assertListEqual(data, natural_sorted(self.corpus.titles)[::-1])
 
+    def test_preprocess_words(self):
+        corpus = Corpus.from_file("book-excerpts")
+        words = [
+            "House",
+            "dóctor",
+            "boy",
+            "way",
+            "Rum https://google.com",
+            "<p>abra<b>cadabra</b><p>",
```

Comment:
If you add `https://google.com` to words, the `_preprocess_words` crashes.

Reply:
Should be fixed too.